source/guides/clear/swupd-3rd-party.rst

```diff
@@ -0,0 +1,273 @@
+All installed 3rd-party bundles reside in :file:`/opt/3rd-party/bundles/<repo-name>/`.
+
+#. First, add a repo link to the web server.
+   The `os-core` bundle will be added automatically when adding a repo.  
+   It contains items that mixer injected into the mix such as version information, 
+   format, CONTENTURL, VERSIONURL, and certificate.
+    
+   .. code-block:: bash
+
+      sudo swupd 3rd-party add my-3rd-party-repo \
+      http://<IP-address-of-web-server> --allow-insecure-http
+
+   .. note::
+      
+      By default, the :command:`swupd` client is designed to communicate 
+      with an HTTPS server. For development purposes, the swupd client 
+      can talk to an HTTP server if you add the flag ``--allow-insecure-http``.
+
+      To avoid adding this flag each time when invoking :command:`swupd`, enter:
+
+      .. code-block:: bash
+
+         sudo mkdir -p /etc/swupd
+
+         sudo tee -a /etc/swupd/config << EOF
+         [GLOBAL]
+         allow-insecure-http=true
+         EOF
+
+#. Query the list of bundles from the repo.
+   
+   .. code-block:: bash
+
+      sudo swupd 3rd-party bundle-list -a 
+
+#. Add the `helloclear` bundle.
+   
+   .. code-block:: bash
+
+      sudo swupd 3rd-party bundle-add helloclear
+
+#. List installed 3rd-party bundles.
+
+   .. code-block:: bash
+
+      sudo swupd 3rd-party bundle-list
+
+#. Look in :file:`/opt/3rd-party` to confirm they were installed there.  
+   
+   .. code-block:: bash
+
+      tree /opt/3rd-party
+
+   Example out:
+   
+   .. code-block:: console
+
+      /opt/3rd-party/
+      ├── bin
+      │   └── helloclear.sh
+      ├── bundles
+      │   └── my-3rd-party-repo
+      │       └── usr
+      │           ├── bin
+      │           │   └── helloclear.sh
+      │           ├── lib
+      │           │   └── os-release
+      │           └── share
+      │               ├── clear
+      │               │   ├── bundles
+      │               │   │   ├── helloclear
+      │               │   │   └── os-core
+      │               │   ├── update-ca
+      │               │   │   └── Swupd_Root.pem
+      │               │   ├── version
+      │               │   └── versionstamp
+      │               └── defaults
+      │                   └── swupd
+      │                       ├── contenturl
+      │                       ├── format
+      │                       └── versionurl
+      └── repo.ini
+
+      12 directories, 12 files
+
+Create more bundles and add to client
+*************************************
+
+From here on, to add new bundles to your mix, follow these steps:
+
+#. Follow the steps above to add a new directory for each app and put content into it.
+
+#. In the mixer workspace, run :command:`mixer versions update`.
+
+#. Follow the remaining mixer process to add and build bundles.  
+
+On the client side:
+
+#. Run :command:`sudo swupd 3rd-party update` to update to the latest version of your mix.
```

Comment:
I know I'm a little late to this review but one thing it may be worth mentioning here is that if autoupdate is enabled in swupd it will automatically update not only the regular swupd upstream content but also the 3rd-party content.